stream_alert/alert_processor/outputs.py

```diff
@@ -0,0 +1,343 @@
+        values_json = json.dumps({
+            'service_key': creds['service_key'],
+            'event_type': 'trigger',
+            'incident_key': kwargs['rule_name'],
+            'description': message,
+            'details': kwargs['alert'],
+            'client': 'StreamAlert'
+        })
+
+        resp = self._request_helper(creds['url'], values_json)
+        success = self._check_http_response(resp)
+
+        self._log_status(success)
+
+
+@output
+class PhantomOutput(StreamOutputBase):
+    """PhantomOutput handles all alert dispatching for Phantom"""
+    __service__ = 'phantom'
+
+    def get_user_defined_properties(self):
+        """Get properties that must be asssigned by the user when configuring a new Phantom
+        output.  This should be sensitive or unique information for this use-case that needs
+        to come from the user.
+
+        Every output should return a dict that contains a 'descriptor' with a description of the
+        integration being configured.
+
+        Phantom also requires a ph_auth_token that represnts an authorization token for this
+        integration and a user provided url to use for alert dispatching. These values should be
+        masked during input and are credential requirements.
+
+        Returns:
+            [OrderedDict] Contains various OutputProperty items
+        """
+        return OrderedDict([
+            ('descriptor',
+             OutputProperty(description='a short and unique descriptor for this '
+                                        'Phantom integration')),
+            ('ph_auth_token',
+             OutputProperty(description='the auth token for this Phantom integration',
+                            mask_input=True,
+                            cred_requirement=True)),
+            ('url',
+             OutputProperty(description='the endpoint url for this Phantom integration',
+                            mask_input=True,
+                            cred_requirement=True))
+        ])
+
+    def _setup_container(self, rule_name, container_url, headers):
+        """Establish a Phantom container to write the alerts to
+
+        Args:
+            rule_name [string]: The name of the rule that triggered the alert
+            container_url [string]: The contructed endpoint url for Phantom containers
+            headers [dict]: A dictionary containing header parameters
+
+        Returns:
+            [integer] ID of the Phantom container where the alerts will be sent
+        """
+        message = 'StreamAlert Rule Triggered - {}'.format(rule_name)
+        ph_container = {'name' : message,
+                        'description' : message}
+        container_string = json.dumps(ph_container)
+        resp = self._request_helper(container_url, container_string, headers, False)
+
+        if not self._check_http_response(resp):
+            self._log_status(False)
+            return False
```

Comment:
The docstring says you return an integer. Either update the docstring or else return a negative number on error